src/turnkeyml/analyze/script.py

```diff
@@ -138,11 +138,6 @@ def explore_invocation(
                     inputs[all_args[i]] = args[i]
         invocation_info.inputs = inputs
 
-    # Save model labels
-    if model_info.model_type != build.ModelType.ONNX_FILE:
```

Comment:
keep THIS if statement

Reply:
solved in https://github.com/onnx/turnkeyml/pull/54/commits/bb65d005cbf567238291fce3828159c219a419b4